src/Pages/SignUp/SignUp.js

```diff
@@ -0,0 +1,381 @@
+  background-image: url("https://images.unsplash.com/8/les-portes.jpg?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=1080&fit=max");
+  background-size: cover;
+`;
+
+const LeftContainer = styled.div`
+  height: 100%;
+  padding: 41px 51px;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+`;
+
+const UpperContent = styled.div`
+  position: absolute;
+  top: 8%;
+`;
+
+const Title = styled.div`
+  font-size: 48px;
+  color: white;
+`;
+
+const SubTitle = styled(Title)`
+  font-size: 24px;
+`;
+
+const Text = styled.div`
+  font-size: 15px;
+  color: black;
+  margin-bottom: 10px;
+
+  .invalid {
+    color: red;
+  }
+`;
+
+const CenterContent = styled.div`
+  position: absolute;
+  top: 42%;
+`;
+
+const BottomContent = styled.div`
+  position: absolute;
+  bottom: 8%;
+  p {
+    font-size: 15px;
+    color: white;
+  }
+`;
+
+const RightBody = styled.div`
+  height: 100%;
+  padding: 120px 115px 20px;
+`;
+
+const RightContainer = styled.div`
+  padding: 0 12px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+`;
+
+const RightTitle = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+  .signUpTitle {
+    font-size: 46px;
+  }
+  .flex {
+    margin-top: 20px;
+    display: flex;
+    justify-content: center;
+
+    .subTitle {
+      font-size: 15px;
+    }
+  }
+`;
+const StyledLink = styled(Link)`
+  font-size: 15px;
+  color: ${(props) => props.theme.colors.grayColor};
+  margin-left: 5px;
+  text-decoration: underline;
+  &:hover {
+    color: black;
+  }
+`;
+
+const Button = styled.button`
+  width: 100%;
+  height: 44px;
+  background-color: rgb(248, 248, 249);
+  padding: 9px 58px 9px 18px;
+  margin: 60px 0 20px;
+  border: 1px solid rgb(221, 224, 226);
+  border-radius: 3px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  overflow: hidden;
+
+  img {
+    width: 80px;
+    margin-right: -20px;
+  }
+`;
+
+const ButtonText = styled.p`
+  font-size: 15px;
+  color: black;
+  text-align: center;
+`;
+
+const InputUpperText = styled.div`
+  display: flex;
+
+  .invalid {
+    color: red;
+  }
+`;
+
+const OrText = styled(Text)`
+  font-size: 12px;
+  text-align: center;
+  margin-bottom: 30px;
+`;
+
+const Input = styled.div`
+  input {
+    width: 100%;
+    height: 44px;
+    margin-bottom: 30px;
+  }
+`;
+
+const SignUpButton = styled(Button)`
+  margin-top: 10px;
+  background-color: black;
+  padding: 9px 18px;
+
+  p {
+    font-size: 15px;
+    color: white;
+  }
+`;
+
+const GreyText = styled(Text)`
+  color: ${(props) => props.theme.colors.grayColor};
+  margin-left: 10px;
+`;
+
+const Flex = styled.div`
+  display: flex;
+  .firstInput {
+    width: 100%;
+  }
+  #marginRight {
+    margin-right: 20px;
+  }
+`;
+
+const LastText = styled(GreyText)`
+  font-size: 13px;
+  text-align: center;
+`;
+
+const FirstInput = styled(Input)`
+  input {
+    width: 100%;
+  }
+`;
```

Comment:
꼭 nesting으로 사용해야 하는 내용인가요? 아래와 같이 사용하면 안되는지 궁금합니다
```suggestion
const FirstInput = styled(Input)`
  width: 100%;
`;
```

```
<FirstInput
 type="text"
  onChange={(e) => handleInput("firstName", e)}
  name="firstName"
  onBlur={isFirstNameValid}
  onKeyUp={isFirstNameValid}
/>
```

Reply:
@saengmotmi  아 종택님 제가 소셜 로그인 하던거 끝내고 커밋하려고 멘트만 달고 수정한걸 커밋을 안했네요 그래서 지금꺼는 처음 커밋 버전이라 수정이 안됏어요 죄송해요 ㅠ 처음 말씀주셨을 때 해당내용은 수정했습니다